docker/web/run_dev_web.sh

```diff
@@ -0,0 +1,3 @@
+pip install django_extensions
```

Comment:
Django extensions should be already installed as they are on `requirements.txt`, why are you adding them here?